tests/providers/google/cloud/triggers/test_cloud_batch.py

```diff
@@ -69,10 +69,10 @@ async def test_trigger_on_success_yield_successfully(
         """
         Tests the CloudBatchJobFinishedTrigger fires once the job execution reaches a successful state.
         """
-        state = JobStatus.State.SUCCEEDED
+        state = JobStatus.State(JobStatus.State.SUCCEEDED)
```

Comment:
Hmm, this seems a bit wrong, the previous code should work.

Reply:
Agree. But it does not :) ... Not sure what else I can do about it :)